src/app/components/login/login.component.ts

```diff
@@ -26,6 +28,34 @@ export class LoginComponent {
 
   matcher = new ErrorMatcher();
 
+  constructor(private loginService: LoginService, private userService: UserService) {}
+
+  sendDecodeFormData() {
+    let uname: string = "";
+    let pass: string = "";
```

Comment:
```suggestion
    let pass = "";
```
No need to declare type when u initialize it